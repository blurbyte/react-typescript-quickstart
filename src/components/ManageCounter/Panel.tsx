import styled from '@styles/styledComponents';
import { Colors } from '@styles/colors';

const Panel = styled.div`
  width: 40rem;
  border-top: 1px solid ${Colors.gray};
  margin-top: 3rem;
`;

export default Panel;
