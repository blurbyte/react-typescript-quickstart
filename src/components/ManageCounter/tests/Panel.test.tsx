import * as React from 'react';
import { shallow } from 'enzyme';

import Panel from '../Panel';

describe('Panel', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Panel />);
    expect(wrapper).toMatchSnapshot();
  });
});
