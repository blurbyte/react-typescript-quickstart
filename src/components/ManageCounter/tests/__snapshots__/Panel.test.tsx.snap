// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel renders correctly 1`] = `
.c0 {
  width: 40rem;
  border-top: 1px solid #908e89;
  margin-top: 3rem;
}

<div
  className="c0"
/>
`;
